office/GanttProject/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           java 1.0
+
+github.setup        bardsoftware ganttproject 370b88ac6023dcc51469406063b0b1292c92cdcf
+
+name                GanttProject
+version             2.8.9-git
+revision            0
+
+java.version        1.8
+java.fallback       openjdk8
+
+
+categories          office java
+platforms           darwin
+maintainers         {gmail.com:giansalvo.gusinu @giansalvo} openmaintainer
+license             GPL-3
+
+description         Project scheduling and management software.
+
+long_description \
+    GanttProject is an open-source desktop project scheduling and \
+    management tool. It is written in Java/Swing. \
+    GanttProject was started by Alexandre Thomas in 2003 and many \
+    people participated in the development since that. \
+    It is translated in more than 20 languages.
+
+homepage            https://www.ganttproject.biz/
+
+checksums           rmd160  f5faf2632cf46dc06cb47032deb249b488c830c5 \
+                    sha256  1792967327b26d7fafc9ce7c3cdd937622b41cd5832bcc4ca2376f6ac9fe248f \
+                    size    15429698
+
+depends_lib-append  port:gradle
+
+use_configure       no
+
+build {
+    set ::env(JAVA_HOME) [exec /usr/libexec/java_home -v ${java.version}]
```

Comment:
The JAVA_HOME variable is set by the java portgroup, so this line should not be required.